include/deal.II/numerics/matrix_creator.templates.h

```diff
@@ -622,7 +620,7 @@ namespace MatrixCreator
     {
       struct Scratch
       {
-        Scratch() = default;
+        Scratch() {}
```

Comment:
This is far from being the most important change in this patch, but why was this done? I suspect this showed up due to a merge conflict resolution since @masterleinad recently switched all of these.

Unless I am missing something we should go with `= default;` to avoid multiple definition problems in the link stage.